### ISSUE TITLE
Changing the python shebang

### ISSUE THREAD

**Issue by rbeyer** [pirl-lpl/hirise-tools#6]
In #3, Chris said:

> At PIRL, python may be python 2 or python 3; python3 always exists and is always python 3. This seems to be how things are done?

Not on my system (obviously), and there will be a lot of variation.  On my system, Python 3 is 'python' and Python 2 is a legacy version that I don't use much, and people should stop using.

Many users will be using conda to install ISIS these days, and may also be using a conda environment, such that their 'default' python will be Python 3.

However, most systems that install python these days will indeed have both a python2 and a python3 alias that should point to the right thing.

So, to sum up: changing the shebang line to python3 is probably the most robust thing.

**Comment by SchallerSpace** [comment]
Ah, fiddlesticks to this inconsistency.
